### DISCUSSION TITLE
[BUG] GitHub crashes  when deleting strikethrough text.

### DISCUSSION THREAD

**Discussion by EscapeAltDelete** [community/community#113381] [Mobile]
As the title states GitHub crashes when trying to delete strikethrough text. 

https://github.com/community/community/assets/96600150/fb9aa5eb-2a3d-42cc-a733-7000397570e6

I have provided a crash-log below.
[GitHub-2024-03-20-004711.txt](https://github.com/community/community/files/14658073/GitHub-2024-03-20-004711.txt)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.154.0 (125806213)
Device: iPhone 12 mini (iOS 17.0)
</details>

**Comment by esahmed21** [comment]
Thank you @EscapeAltDelete for reaching out to us. I'm sorry to hear that the app crashes when attempting to delete the text. I'll ensure that an internal ticket is created to investigate and resolve this issue promptly. Thank you for bringing it to our attention.
